From a2a6377d750ec7cef0953daca06e701212f608a7 Mon Sep 17 00:00:00 2001
From: Robby Cai <r63905@freescale.com>
Date: Wed, 11 Mar 2015 20:37:25 +0800
Subject: [PATCH 0214/1221] MLK-10384-2 ARM: dts: add PXP support on i.MX7D
 SDB

Add PxP support on i.MX7D SabreSD board

Signed-off-by: Robby Cai <r63905@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx7d-sdb.dts |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d-sdb.dts b/arch/arm/boot/dts/imx7d-sdb.dts
index 736ebbc..903f04b 100644
--- a/arch/arm/boot/dts/imx7d-sdb.dts
+++ b/arch/arm/boot/dts/imx7d-sdb.dts
@@ -106,6 +106,10 @@
         status = "disabled";
 };
 
+&epxp {
+	status = "okay";
+};
+
 &fec1 {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_enet1>;
-- 
1.7.5.4

